commands/account/auth.ts

```diff
@@ -105,10 +107,20 @@ export async function handler(
 
   const deprecatedConfigExists = configFileExists(false);
   const globalConfigExists = configFileExists(true);
+  let migrationConfirmed;
+  let mergeConfirmed;
   if (deprecatedConfigExists) {
     if (globalConfigExists) {
       try {
-        await handleMerge(providedAccountId);
+        mergeConfirmed = await handleMerge(providedAccountId);
+        if (!mergeConfirmed) {
+          logger.log(
+            i18n('commands.account.subcommands.auth.errors.mergeNotConfirmed', {
+              deprecatedConfigPath: getConfigPath('', false)!,
+              globalConfigPath: getConfigPath('', true)!,
+            })
+          );
+        }
```

Comment:
The nesting is starting to get a little deep here.  Could we split this into smaller functions?